src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -445,8 +554,26 @@ public float Derivative(float output, float label)
     }
 
     /// <summary>
-    /// Tweedie loss, based on the log-likelihood of the Tweedie distribution.
+    /// Tweedie loss, based on the log-likelihood of the Tweedie distribution. This loss function is used in Tweedie regression.
     /// </summary>
+    /// <remarks type="text/markdown"><![CDATA[
+    ///
+    /// The Tweedie Loss function is defined as:
+    ///
+    /// \[
+    /// L(f(\vec{ x}, y, i) =
+    /// \begin{cases}
```

Comment:
@natke @sfilipi do you know if this will render? This is using the amsmath LaTeX package.

Reply:
I think it should be ok @najeeb-kazmi. The only thing we've had trouble with so far is \boldsymbol. We can test it before you merge this PR if you like

Reply:
@natke yes let's test it please before I merge. Let me know if there are any errors. Thanks!

Reply:
Preview looks good: https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers.logloss?branch=smoke-test-preview&view=ml-dotnet

Reply:
@natke my question was regarding SmoothedHingeLoss and TweedieLoss. I don't see the new documentation for those on the site.